adhocracy4/comments_async/static/comments_async/comment.jsx

```diff
@@ -24,7 +25,9 @@ const translated = {
   readMore: django.gettext('Read more...'),
   readLess: django.gettext('Read less'),
   share: django.gettext('Share'),
-  report: django.gettext(' Report')
+  report: django.gettext(' Report'),
+  showModStatement: django.pgettext('kosmo', 'Show moderator\'s statement'),
+  hideModStatement: django.pgettext('kosmo', 'Hide moderator\'s statement')
```

Comment:
Uh, ehm, actually, I think, as this will also end up an a+ and thus in kosmo in the locale-source branch, we should not mark this as translation from komo. While getting the translations will be a longer way now, I think that will save us from confusion at a later point. 
So, let's:
1. remove the stuff marking it for kosmo
2. Also update a4 in a+, push strings to transifex, translate, pull. (That way we will get the translations with the next rebase.)
3. Probably test this story without translations, but add a reminder about that to the release story